pwa/src/templates/templateParts/download/DownloadTemplate.module.css

```diff
@@ -3,9 +3,10 @@
   max-width: var(--web-app-download-popup-width);
   top: 50%;
   left: 50%;
-  transform: translate(-50%, -50%);
   z-index: 700;
-  outline: 0;
+  margin: 0 auto;
+  margin-left: -250px;
+  margin-top: -250px;
```

Comment:
Nice. I do think however, that we should make these margins relative to the `--web-app-download-popup-width`. You're already doing it, but implicitly. 

Making it like this, ensures that when the width is of the pop-up is changed, its still being centered;

```
margin-left: calc(var(--web-app-download-popup-width) / 2 * -1);
margin-top: calc(var(--web-app-download-popup-width) / 2 * -1);
```

Reply:
Edit: the rules could be `margin-inline-start` and `margin-block-start` instead of left and top 👍🏼